pkg/gui/controllers/branches_controller.go

```diff
@@ -810,7 +810,7 @@ func (self *BranchesController) createPullRequestMenu(selectedBranch *models.Bra
 
 	menuItems = append(menuItems, menuItemsForBranch(selectedBranch)...)
 
-	return self.c.Menu(types.CreateMenuOptions{Title: fmt.Sprintf(self.c.Tr.CreatePullRequestOptions), Items: menuItems})
+	return self.c.Menu(types.CreateMenuOptions{Title: fmt.Sprint(self.c.Tr.CreatePullRequestOptions), Items: menuItems})
```

Comment:
I hope this thing checks the value of CreatePullRequestOptions cos for some of these we have a '%s' in there

Reply:
No it doesn't. It simply warns when you are using printf with only a format string, without additional args.

It can (and does) check for '%s' only if the format string is a literal.